packages/generated/client-types/src/lib/index.ts

```diff
@@ -0,0 +1,5 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export * from "./models"
```

Comment:
question: Would it make sense in the end if we have multiple export from here, to namespace them like `export * as Models` so that when you import a type you now what it reference e.g `Models.Address`. It would avoid any collision as well if it happen that we attribute the same name in a different place. wdyt? This would be more of a convention like I am suggesting for medusa/types

Reply:
One of the tenants for this initiative was to be a drop-in replacement of the current type imports from `@medusajs/medusa` in client code. I would not impose a namespace on end users. `import * as Models` can be used if need be.

Reply:
If we export something else than the models later, the user will not be able to do as Models as it would also includes the rest. I agree that it is a drop in replacement so let’s not take this comment into account for now 👍